website_event_sale_check_order_before_payment/static/src/js/event_payment_form.js

```diff
@@ -0,0 +1,33 @@
+odoo.define(
+    "website_event_sale_check_order_before_payment.event_payment_form",
+    function(require) {
+        "use strict";
+
+        var paymentForm = require("payment.payment_form");
```

Comment:
```suggestion
        const paymentForm = require("payment.payment_form");
```

nitpicking: `var` is old-js by convention